chapters/develop-organize/slides/reading/read.md

```diff
@@ -22,58 +22,215 @@ Our choice of engine for storing and rendering slides is [`reveal-md`](https://g
 
-output files are:
+If a slide deck is created from slides from multiple topics, then the `Makefile` and `slides.mdpp` will be placed in the chapter directory.
+As an example, see the contents for the [`operating-systems` repository](https://github.com/open-education-hub/operating-systems/tree/main/content/software-stack):
+
+```text
+|-- Makefile
+|-- overview/
+|   |-- media/
+|   `-- slides/
+|-- software-types/
+|   |-- media/
+|   `-- slides/
+|-- operating-system/
+|   |-- media/
+|   `-- slides/
+|-- operating-system-types/
+|   |-- media/
+|   `-- slides/
+|-- modern-software-stacks/
+|   |-- media/
+|   `-- slides/
+`-- slides.mdpp
+```
+
+with `slides.mdpp` referencing the actual Markdown contents:
+
+```text
+---
+title: "OS: Software Stack"
+revealOptions:
+  background-color: 'aquamarine'
+  transition: 'none'
+  slideNumber: true
+  autoAnimateDuration: 0.0
+---
+
+# Software Stack
+
+1. [The Software Stack](#software-stack)
+1. [Types of Software](#types-of-software)
+1. [The Operating System](#operating-systems)
+1. [Types of Operating Systems](#types-of-operating-systems)
+1. [The Software Stack in Modern Systems](#modern-software-stacks)
+
+!INCLUDE "overview/slides/software-stack.md"
+!INCLUDE "software-types/slides/types-of-software.md"
+!INCLUDE "operating-system/slides/operating-system.md"
+!INCLUDE "operating-system-types/slides/os-types.md"
+!INCLUDE "modern-software-stacks/slides/modern-software-stacks.md"
+```
 
-- `slides.md`: intermediary aggregated output file (after `markdown-pp` processing)
-- `slides.pdf`: PDF output file
-  to be viewed with PDF reader
-- `_site/`: HTML output
-  open `_site/index.html`
-  to be viewed with web browser
+## Slide Contents
 
-## Presenting Slides
+Actual slide contents are located in Markdown files for each topic.
+A specialized index file, such as the `slides.mdpp` file above, references actual slide contents for the desired structure.
+Typically, educators would be required to only alter the `slides.mdpp` file to select and order slides according to their needs and preferences.
 
-2D view of slides for `markdown-pp`
+An index file, as above, would consist of:
 
-keyboard shortcuts
+- references to slide sections, as in the line `[The Software Stack](#software-stack)`
+- inclusion of actual slide contents files, as in the line `!INCLUDE "overview/slides/software-stack.md"`
 
-## Slide Contents
+Each actual content slide Markdown file will feature information related to the topic, using the features and syntax of `reveal-md` / `reveal.js`.
+Depending on the content, there are some common type of slides:
+
+- text slides: consisting only / mostly of text items, that may be structured as bullets
+- title slides: consisting only of a title, or one line, to introduce a new topic
+- code snippet slides: consisting of sequence of code to demonstrate a concept
+- media slides: consisting of images, videos or audio items
+- table slides: consisting of a table or a comparison of ideas
```

Comment:
You can also combine these to a certain extent, such as having an image and 2-3 bullets. I wouldn't categorise them so granularly. Maybe rephrase these as types of content that can be present on slides.